### DISCUSSION TITLE
"schedule" jobs runs 7-8 hours after the scheduled time

### DISCUSSION THREAD

**Discussion by yashrathi-git** [community/community#26708] [Actions]
<p><img src="https://user-images.githubusercontent.com/57002207/181099329-abc7117e-6796-4d31-bb2f-78d010f6aaf4.png" alt="image" data-base62-sha1="7j5WWU8pe3JHH49FaeCFVb1XXPm" width="204" height="80"><br>
I have scheduled this workflow to run everyday at 1 AM but gets executed around 8:20-8:30 AM.</p>
<p>I understand that there could be a time lag, but don’t expect that big time lag especially when I just run them once per day.</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hey <a class="mention" href="/yashrathi-git">@yashrathi-git</a>, welcome to the GitHub community!</p>
<p>Scheduled workflows run on UTC times, not your local timezone. I’m not sure where you are but could it be a 7 hour time difference from UTC? If so you might need to convert your cron to work with UTC times instead of your local time zone.</p>

**Comment by yashrathi-git** [comment]
<p>Hey <a class="mention" href="/thomasshaped">@ThomasShaped</a><br>
that’s what I suspected might be happening. So UTC 1 AM is 6:30 IST. So there’s still around 1.5-2hours lag, but 2 hour lag won’t matter for my use-case. Though sometimes the action doesn’t executes at all.<br>
Thanks for the solution though!</p>
